.github/workflows/release.yml

```
@@ -0,0 +1,74 @@
+          push: true
+          tags: |
+            lohanjacobs/vercel-admin-mcp:${{ steps.version.outputs.version }}
+            lohanjacobs/vercel-admin-mcp:latest
+          labels: |
+            org.opencontainers.image.title=Vercel MCP Server
+            org.opencontainers.image.description=Dockerized Vercel MCP Server
+            org.opencontainers.image.version=${{ steps.version.outputs.version }}
+            org.opencontainers.image.source=${{ github.server_url }}/${{ github.repository }}
+            org.opencontainers.image.revision=${{ github.sha }}
+          cache-from: type=gha
+          cache-to: type=gha,mode=max
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)